packages/cli/package.json

```diff
@@ -32,8 +32,7 @@
     "prebuild": "rimraf dist",
     "build": "tsc",
     "test": "mocha --timeout 60000 -r ts-node/register 'src/**/**.test.ts'",
-    "coverage": "nyc npm run test",
-    "prepare": "npm run build"
```

Comment:
Talked with @michael-siek and this isn't needed. When left in it tries to build the cli before it links packages together, which would cause the build to fail as @axe-core/webdriverjs wasn't available yet.